Sources/GRPC/ClientConnection.swift

```diff
@@ -576,3 +576,16 @@ extension HTTP2ToHTTP1ClientCodec.HTTPProtocol {
     }
   }
 }
+
+fileprivate extension String {
```

Comment:
Can you reformat this to match the style used elsewhere?

Reply:
Tip: `make project` will automatically configure the Xcode project to use the correct indentation.